### PR TITLE
locale: add force file reload and add translation functions

### PR DESCRIPTION
**Pull request by jokaio**
force file reload: translation files are reloaded even if they are marked as loaded
add translation: a wrapper for the AddTranslation function of the i18n bundle

### CODE REVIEW

**Comment by danielpoe** [comment]
> force file reload: translation files are reloaded even if they are marked as loaded
> add translation: a wrapper for the AddTranslation function of the i18n bundle

Why and where is this needed?
I remember idea was to reload files if debug mode is on.

File: core/locale/infrastructure/translationService.go [hunk 1]
```diff
@@ -4,6 +4,8 @@ import (
 	"bytes"
 	"errors"
 	"fmt"
+	"github.com/nicksnyder/go-i18n/i18n/language"
```
Comment (bastianccm) [inline core/locale/infrastructure/translationService.go]: Please run `goimports` to fix the import ordering

**Comment by bastianccm** [comment]
@danielpoe this is to allow multiple translations/translation-sources and let other projects add and remove custom translations on the fly

**Comment by danielpoe** [comment]
> @danielpoe this is to allow multiple translations/translation-sources and let other projects add and remove custom translations on the fly

Still dont get the use case by the descriptions in this PR.

My thoughts:
So an external package uses the interface layer directly? From which layer?
Methods like ForceFileReload are very implementation specific.

Why not implement your own TranslationService Adapter in the project (that may aggregate the default one)

**Pull request closed without merging**